adagrams/game.py

```diff
@@ -1,11 +1,82 @@
+import random
+import string
+
+value_of_letters = {
```

Comment:
This variable is a constant. Its values don't change and they're not supposed to. The convention for naming constant variables is to use all caps so we can rename this as `VALUE_OF_LETTERS`